patterns/definitions.owl

```diff
@@ -27996,10 +28206,10 @@ AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.ob
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/XPO_0103556> "abnormal anatomical structure morphogenesis in optic vesicle"^^xsd:string)
 EquivalentClasses(<http://purl.obolibrary.org/obo/XPO_0103556> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0001236> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000052> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/GO_0009653> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000066> <http://purl.obolibrary.org/obo/XAO_0000228>))) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
 
-# Class: <http://purl.obolibrary.org/obo/XPO_0103557> (abnormal anatomical structure morphogenesis in neural fold)
+# Class: <http://purl.obolibrary.org/obo/XPO_0103557> (abnormal anatomical structure morphogenesis in neural fold (obsolete))
 
```

Comment:
Does this make sense? @seger

Reply:
No. Let me look into it..

Reply:
Did you find out what happened?

Reply:
I used the wrong ID for neural fold in src/patterns/data/manual/abnormalBiologicalProcessInLocation.tsv at line 88. It should be XAO:0004087. @matentzn